Upgrade to latest release of reprocessing and bsb-native

bsansouci opened the pull request:
:)

bsansouci committed:
Upgrade to latest release of reprocessing and bsb-native

cnguy commented:
@bsansouci I'll play around with this again! Still just thinking of an idea. >//< I see you really want everyone to update their dependencies!

edit: Thanks!

bsansouci commented:
Hah yeah, we published a version of reprocessing and updated master, so your project would've been broken if I hadn't updated it :p 
So I updated everyone on github
